doc/whats_new.rst

```diff
@@ -169,8 +173,8 @@ NEW
 - :func:`nilearn.datasets.load_mni152_wm_mask` loads mask from the white-matter
   MNI152 template.
 - :func:`nilearn.image.binarize_img` binarizes images into 0 and 1.
-- :class:`nilearn.input_data.NiftiMasker`,
-  :class:`nilearn.input_data.MultiNiftiMasker`, and objects relying on such maskers
+- :class:`nilearn.maskers.NiftiMasker`,
```

Comment:
shouldn't we update the name just for the whatsnew of the next release?

Reply:
Yes, I hesitated between de-referencing the old entries and updating the old references. I ended up doing the latter mostly for user convenience and also because entries are usually not related to where the maskers live but to functionalities of the objects. I agree that doing so modify the "history" and conveys the impression that maskers were always in the "maskers" module.
I don't have a strong preference, so we can go with the former if you think this is better.

Reply:
I see, that makes sense! we can keep it like this then